docs/content/contribute/tutorial.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Contributing Tutorial
+---
+
+We are excited to have you join the Porter community! This tutorial will walk 
+you through how to modify Porter and try out your changes. We encourage you to
+follow the tutorial before submitting your first pull request because this will
+help you get your environment set up and become familiar with how to make a change
+and test it out.
+
+If you run into any trouble, please let us know! The best way to ask for help is
+to start a new discussion on our [forum].
+
+{{< toc >}}
+
+[forum]: /forum/
+
+## Try Porter
+
+Before contributing to any open-source project, you should start with becoming
+familiar with the software first. 
+
+1. [Install Porter](/install/)
+1. Run through the [QuickStart]
+
+After this you should be able to answer:
+
+* What is Porter?
+* What is a bundle?
+* How do I make a bundle?
+* How do I install a bundle?
+
+We are always iterating on our documentation. If something isn't clear and you
+have read our [documentation], let us know by opening an issue. Say what you
+were trying to learn about, where you looked, if you could find a relevant page,
+and what still isn't clear. We can then answer your question and improve our docs
+so the next person has a better experience.
+
+[documentation]: /docs/
+
+## Setup Environment
+
+First let's get your computer setup so that you can work on Porter. You will
+need a few things.
+
+We are improving our support for building Porter on Windows. In a few weeks, we
+will have it all working on any Windows shell. For now, if you are on Windows,
+please use [Windows Subsystem for Linux][wsl] (WSL). The rest of this tutorial
+assumes that you are inside your WSL distribution when installing prerequisites
+and executing the commands.
+
+* [Git]
+   
+   If you are new to Git or GitHub, we recommend reading the [GitHub Guides].
+   They will walk you through installing Git, forking a repository and
+   submitting a pull request.
+* [Go](https://golang.org/doc/install) version 1.13 or higher
+* [Docker]
+
+[GitHub Guides]: https://guides.github.com/
+[Git]: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+[wsl]: https://docs.microsoft.com/en-us/windows/wsl/install-win10
+[Docker]: https://docs.docker.com/get-docker/
+
+## Checkout Code
+
+Porter can either be cloned into your GOPATH (usually ~/go) or anywhere on your computer.
+
+Porter has a couple repositories. You may want to clone Porter and its other
```

Comment:
nit: There are more than (the strict definition of) a couple repos... Also, worth mentioning the general theme of associated repos (mixins, plugins, etc.)... and/or maybe link to the [getporter org](https://github.com/getporter) for a quick overview of all of them?